numenc-cpp/encoder_decoder.cpp

```diff
@@ -935,13 +912,13 @@ static PyMethodDef EncdecMethods[] = {
 
 static struct PyModuleDef cModPyDem = {
     PyModuleDef_HEAD_INIT,
-    "numenc_module",
-    "fast number encoding and decoding in C",
+    "numenc",
+    "Encode and decode numbers to sortable bytes in C++",
```

Comment:
"in C++" is probably not relevant part of the module documentation, so I would drop it.